src/encoded/types/file.py

```diff
@@ -163,9 +164,23 @@ def post_upload(context, request):
     return CorePostUpload(context, request)
 
 
+def validate_user_has_protected_access(request):
+    """ Validates that the user who executed the request context either is
+        an admin or has the dbgap group
+    """
+    principals = request.effective_principals
+    if 'group.admin' in principals or 'group.dbgap' in principals:
+        return True
+    else:
+        return False
```

Comment:
Consider simplifying.
```suggestion
    return False
```